src/prediqt.ts

```diff
@@ -3,7 +3,12 @@ import {SignatureProvider} from "eosjs/dist/eosjs-api-interfaces";
 const fetch = require("isomorphic-fetch");
 
 import {TransactParams, Authorization, Balance, Fee, Market, Order, Share, TransferShares} from "./interfaces/prediqt";
-import {isObject} from "./utils";
+import {isObject, processData} from "./utils";
+
+enum  CancelOrderNameId  {
```

Comment:
probably we could use a more generic name for this, since we might use it in other actions something like:

OrderType or Outcome or Result